nin/backend/utils.js

```diff
@@ -50,10 +51,10 @@ function mergeOptions(input, defaults) {
 function colorizeCommanderHelpText(txt) {
   return (txt.replace(/(\[[^\]]*\])/g, chalk.grey('$1'))
              .replace(/Usage: (\w+)/g, 'Usage: ' + chalk.green('$1'))
-             .replace(/(\n  \w+ [^\n]*)/g, chalk.cyan('$1'))
-             .replace(/(\n   ( -+[\w,-]+)+)/g, chalk.green('$1'))
-             .replace(/(\n    \w+)/g, chalk.green('$1'))
-             .replace(/(\n  \w+:)/g, chalk.yellow('$1')));
+             .replace(/(\n{2}\w+ [^\n]*)/g, chalk.cyan('$1'))
```

Comment:
Did eslint suggest these changes? They are not equivalent. Before:

![image](https://cloud.githubusercontent.com/assets/578029/20238794/4af1939c-a8f3-11e6-9226-a64c38931bf0.png)

After:

![image](https://cloud.githubusercontent.com/assets/578029/20238799/5cffe9c6-a8f3-11e6-89ca-64c229ec6667.png)